src/client/sandbox/index.js

```diff
@@ -160,7 +160,7 @@ export default class Sandbox extends SandboxBase {
 
         this.event.initDocumentListening();
 
-        if (isWebKit)
+        if (window.top === window.self && isWebKit)
```

Comment:
We should not use window.self because of https://github.com/DevExpress/testcafe/issues/632

@AndreyBelym it's enough to use `window.top === window` because no one can override `window` in our code
@LavrovArtem maybe we should add this rule to eslint-plugin-hammerhead?